git/git_test.go

```diff
@@ -325,6 +326,76 @@ func TestListRemoteWithMultipleBranches(t *testing.T) {
 	assertEqualRemotes(t, got, want)
 }
 
+func TestShowMetadata(t *testing.T) {
+	type testcase struct {
+		name        string
+		title       string
+		description string
+	}
+
+	tests := []testcase{
+		{
+			name:        "title-only",
+			title:       "add feature x",
+			description: "",
+		},
+		{
+			name:        "title with description",
+			title:       "add feature y",
+			description: "this is the latest feature",
+		},
```

Comment:
I think makes sense to add a case here with a multiline body (I think this is not covered).